syntax = "proto3";

service HelloService {
  rpc SayHello (HelloRequest) returns (HelloReply) {}

}

message HelloRequest {
  string name = 1;
}

message HelloReply {
  string message = 1;
}



